grype/presenter/cyclonedx/test-fixtures/snapshot/TestCycloneDxPresenterDir.golden

```diff
@@ -26,14 +26,18 @@
           "name": "syft:package:type",
           "value": "rpm"
         },
+        {
```

Comment:
Should we have a snapshot test that asserts the right value ends up here? I was surprised to see this being set to `""` in both snapshots for this PR.

Reply:
that's a good point, let me update the test packages to have a metadata object (thus they'll have a metadata type specified)

Reply:
actually, there is a metadata type, let me dig more I think something might be broken

Reply:
Ok, the test fixture was using grype packages as the basis, not syft packages. I've updated the fixture now and this issue should be resolved.